src/test/sushi-integration.t.sol

```diff
@@ -278,26 +321,26 @@ contract SushiIntegrationTest is TestBase {
                 uint256 newCrops = rmul(pstake, pshare + rdiv(punclaimedRewards, ptotal));
                 if (newCrops > pcrops) sushiToUser = newCrops - pcrops;
             }
-            assertEq(usr.sushi(), psushi + sushiToUser);
+            assertEqApprox(usr.sushi(), psushi + sushiToUser, 1);
             if (join.total() > 0) {
-                assertEq(join.stock(), pstock + punclaimedRewards - sushiToUser);
+                assertEqApprox(join.stock(), pstock + punclaimedRewards - sushiToUser, 1);
             } else {
                 assertTrue(join.stock() <= dust);  // May be a slight rounding error
             }
             if (ptotal > 0) {
-                assertEq(join.share(), pshare + rdiv(punclaimedRewards, ptotal));
+                assertEqApproxBPS(join.share(), pshare + rdiv(punclaimedRewards, ptotal), 500);
```

Comment:
500 bps is pretty big (5%), but as noted right now the `assertEqApproxBPS` is using 10^5 instead of 10^4, so this is currently a 0.5% tolerance; make sure to update this and other invocations once `assertEqApproxBPS` is fixed.